content/applications/productivity/knowledge/management.rst

```diff
@@ -2,113 +2,240 @@
+   and `Members` in the :guilabel:`Visibility`. Invite users and select `Can edit` as their
+   specific access right. The invited users can access and edit the article in the
+   :guilabel:`Workspace` category. Other members must hover over the :guilabel:`Workspace`
+   category, click the :icon:`fa-eye` (:guilabel:`eye`) icon to search for it in the hidden
+   articles, and click :guilabel:`Join` to edit the article.
+
+.. _knowledge/management/share-online:
 
-Creation
---------
+Share online
+------------
 
-To create articles, click the :guilabel:`+ New` button on the right side of the top bar or the
-:guilabel:`+` button next to a category or another article.
+To share an article online, click :guilabel:`Share` and activate the :guilabel:`Share to web`
+button to generate a URL link. Click the :icon:`fa-clone` (:guilabel:`copy`) icon to copy the link
+and share it with external users.
 
-.. tip::
-   Create private articles quickly with the Alt/Option + C keyboard shortcut.
+.. note::
+   If an article contains :ref:`inserted views <knowledge/articles_editing/views>`, users with the
+   link will not see them unless they can access the inserted content.
 
-.. _management/share:
+.. _knowledge/management/remove:
 
-Sharing
--------
+Article removal
+===============
 
-You can share articles with internal or external users. To do so, open the share menu by clicking
-:guilabel:`Share` in the top-right menu of articles.
+Removing an article involves deleting or archiving it:
 
-Invite users
-~~~~~~~~~~~~
+- **Delete an article**
 
-To share articles with specific users (internal or external, such as a partner or a customer), click
-:guilabel:`Invite`. This opens a pop-up window in which you can choose the :guilabel:`Permission`
-:ref:`(i.e, access rights) <management/categories>` and enter the :guilabel:`Recipients`' name or
-email.
+Select an article in the sidebar tree, and click the :icon:`fa-ellipsis-v` (:guilabel:`ellipsis`)
+icon, then :guilabel:`Send to Trash`. The article is moved to the trash for 30 days before being
+permanently deleted.
 
-You can also restrict a specific user from accessing the article by selecting :guilabel:`No access`
-permission.
+Alternatively, click :guilabel:`Search` in the top-left menu. Select an article and click
+:menuselection:`Actions --> Delete --> Delete` to remove the article permanently.
 
-.. image:: management/invite.png
-   :alt: pop-up window to invite users to access a Knowledge article
+- **Archive an article**
 
-Share online
-~~~~~~~~~~~~
+Click :guilabel:`Search`, select an article, and click :menuselection:`Actions --> Archive -->
+Archive`.
 
-To share articles **online**, activate the :guilabel:`Share to web` button. Doing so generates a URL
-link anyone can use to view the article.
+.. note::
+   - By default, archived articles have a 30-day expiration date before being permanently deleted.
```

Comment:
Are you sure archived articles are deleted? Archived records are not deleted usually, and I can't see a deletion date like for trashed articles